aws/createBlueprint.sh

```diff
@@ -30,17 +30,15 @@ DESCRIPTION:
         ( tenant, account, product, solution, environment, segment )
 
 USAGE:
-  $(basename $0) -i GENERATION_INPUT_SOURCE -t GENERATION_TESTCASE -s GENERATION_SCENARIOS
+  $(basename $0) -i GENERATION_INPUT_SOURCE -s GENERATION_SUBPROVIDERS
 
 PARAMETERS:
 
 (o) -i GENERATION_INPUT_SOURCE is the source of input data to use when generating the template
     -h                         shows this text
 (o) -o OUTPUT_DIR              is the directory where the outputs will be saved - defaults to the PRODUCT_STATE_DIR
-(o) -p GENERATION_PROVIDER     is the provider to for template generation
+(o) -p GENERATION_PROVIDERS     is the provider to for template generation
```

Comment:
Normally we just allow the switch multiple times and gather the values e.g. -v switches for freemarker.sh

Reply:
Fixed up now accepts a list of -p values or an env var `GENERATION_PROVIDERS`